configure.py

```diff
@@ -105,6 +106,60 @@ def _gen_android_cross_file(cfg):
         fp.write(cross_file)
 
 
+_IOS_ABI_MAP = dict(
+    aarch64="arm64",
+    x86_64="x86_64",
+)
+
+
+_IOS_CROSS_FILE_TPL = textwrap.dedent(
```

Comment:
Make sure the existing ini files are dropped

Reply:
Done.